<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
<meta name="pinterest" content="nopin">
<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
<meta name="generator" content="Hugo 0.36" />



<link rel="canonical" href="/post/2011-10-21-copy/">


    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet">
    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/solarized_dark.min.css">
    <title>William Astle: Bayesian modelling of GWAS data using linear mixed models - My New Hugo Site</title>
    
<meta name="description" content="Date: 2011-10-21 Time: 15:30-16:30 Location: BURN 1205 Abstract: Genome-wide association studies (GWAS) are used to identify physical positions (loci) on the genome where genetic variation is causally associated with a phenotype of interest at the population level. Typical studies are based on the measurement of several hundred thousand single nucleotide polymorphism (SNP) variants spread across the genome, in a few thousand individuals. The resulting datasets are large and require computationally efficient methods of statistical analysis.">

<meta property="og:title" content="William Astle: Bayesian modelling of GWAS data using linear mixed models - My New Hugo Site">
<meta property="og:type" content="article">
<meta property="og:url" content="/post/2011-10-21-copy/">
<meta property="og:image" content="/images/default.png">
<meta property="og:site_name" content="My New Hugo Site">
<meta property="og:description" content="Date: 2011-10-21 Time: 15:30-16:30 Location: BURN 1205 Abstract: Genome-wide association studies (GWAS) are used to identify physical positions (loci) on the genome where genetic variation is causally associated with a phenotype of interest at the population level. Typical studies are based on the measurement of several hundred thousand single nucleotide polymorphism (SNP) variants spread across the genome, in a few thousand individuals. The resulting datasets are large and require computationally efficient methods of statistical analysis.">
<meta property="og:locale" content="ja_JP">

<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="My New Hugo Site">
<meta name="twitter:url" content="/post/2011-10-21-copy/">
<meta name="twitter:title" content="William Astle: Bayesian modelling of GWAS data using linear mixed models - My New Hugo Site">
<meta name="twitter:description" content="Date: 2011-10-21 Time: 15:30-16:30 Location: BURN 1205 Abstract: Genome-wide association studies (GWAS) are used to identify physical positions (loci) on the genome where genetic variation is causally associated with a phenotype of interest at the population level. Typical studies are based on the measurement of several hundred thousand single nucleotide polymorphism (SNP) variants spread across the genome, in a few thousand individuals. The resulting datasets are large and require computationally efficient methods of statistical analysis.">
<meta name="twitter:image" content="/images/default.png">


<script type="application/ld+json">
  {
    "@context": "http://schema.org",
    "@type": "NewsArticle",
    "mainEntityOfPage": {
      "@type": "WebPage",
      "@id":"/"
    },
    "headline": "William Astle: Bayesian modelling of GWAS data using linear mixed models - My New Hugo Site",
    "image": {
      "@type": "ImageObject",
      "url": "/images/default.png",
      "height": 800,
      "width": 800
    },
    "datePublished": "2011-10-21T00:00:00JST",
    "dateModified": "2011-10-21T00:00:00JST",
    "author": {
      "@type": "Person",
      "name": "My New Hugo Site"
    },
    "publisher": {
      "@type": "Organization",
      "name": "My New Hugo Site",
      "logo": {
        "@type": "ImageObject",
        "url": "/images/logo.png",
        "width": 600,
        "height": 60
      }
    },
    "description": "Date: 2011-10-21 Time: 15:30-16:30 Location: BURN 1205 Abstract: Genome-wide association studies (GWAS) are used to identify physical positions (loci) on the genome where genetic variation is causally associated with a phenotype of interest at the population level. Typical studies are based on the measurement of several hundred thousand single nucleotide polymorphism (SNP) variants spread across the genome, in a few thousand individuals. The resulting datasets are large and require computationally efficient methods of statistical analysis."
  }
</script>


    <link href="/css/styles.css" rel="stylesheet">
    

  </head>

  <body>
    
    
    

    <header class="l-header">
      <nav class="navbar navbar-default">
        <div class="container">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">My New Hugo Site</a>
          </div>

          

        </div>
      </nav>
    </header>

    <main>
      <div class="container">
        
<div class="row">
  <div class="col-md-8">

    <nav class="p-crumb">
      <ol class="breadcrumb">
        <li><a href="/"><i class="fa fa-home" aria-hidden="true"></i></a></li>
        
        <li itemscope="" itemtype="http://data-vocabulary.org/Breadcrumb"><a href="/post/" itemprop="url"><span itemprop="title">post</span></a></li>
        
        <li class="active">William Astle: Bayesian modelling of GWAS data using linear mixed models</li>
      </ol>
    </nav>

    <article class="single">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2011-10-21T00:00:00JST">Oct 21, 2011</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="/post/">post</a></li>
      
    </ul>

    <h1 class="title">William Astle: Bayesian modelling of GWAS data using linear mixed models</h1>
  </header>

  

  <div class="article-body">

<h4 id="date-2011-10-21">Date: 2011-10-21</h4>

<h4 id="time-15-30-16-30">Time: 15:30-16:30</h4>

<h4 id="location-burn-1205">Location: BURN 1205</h4>

<h2 id="abstract">Abstract:</h2>

<p>Genome-wide association studies (GWAS) are used to identify physical positions (loci) on the genome where genetic variation is causally associated with a phenotype of interest at the population level. Typical studies are based on the measurement of several hundred thousand single nucleotide polymorphism (SNP) variants spread across the genome, in a few thousand individuals. The resulting datasets are large and require computationally efficient methods of statistical analysis.</p>

<p>Two variance components linear mixed models have recently been proposed as a method of analysis for GWAS data that can control for the confounding effects of population stratification, by modelling the correlation between study subjects induced by relatedness. Unfortunately, standard methods for fitting linear mixed models are computationally intensive because computation of the likelihood depends on the inversion of a large matrix which is a function of the model parameters.  I will describe a fast method for calculating the likelihood of a two variance components linear model which allows analysis of a large GWAS dataset using mixed models by Bayesian inference. A Bayesian analysis of GWAS provides a natural way of overcoming the so-called &ldquo;multiple-testing&rdquo; problem which arises from the large dimension of the predictor variable space. In the Bayesian framework we should have low prior belief that any particular genetic variant explains a large proportion of the phenotypic variation. The normal-exponential-gamma prior as been proposed as a good representation of such belief and I will describe an efficient MCMC algorithm which allows to incorporate this prior into the modelling.</p>

<h2 id="bio">Bio</h2>

<p><em>William Astle</em> is a Postdoctoral Fellow at McGill University, working with Aur√©lie Labbe and David A. Stephens. He holds a Ph.D. from Imperial Colledge, London.</p>
</div>

  <footer class="article-footer">
    
    
    
    <section class="bordered">
      <header>
        <div class="panel-title">CATEGORIES</div>
      </header>
      <div>
        <ul class="p-terms">
          
          <li><a href="/categories/mcgill-statistics-seminar/">McGill Statistics Seminar</a></li>
          
        </ul>
      </div>
    </section>
    
    
    
    <section class="bordered">
      <header>
        <div class="panel-title">TAGS</div>
      </header>
      <div>
        <ul class="p-terms">
          
          <li><a href="/tags/2011/">2011</a></li>
          
        </ul>
      </div>
    </section>
    
    
  </footer>

</article>


    
  </div>

  <div class="col-md-4">
    
<aside class="l-sidebar">

  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">LATESTS</div>
    </div>
    <div class="list-group">
      
      <a href="/post/2011-10-21-copy/" class="list-group-item">William Astle: Bayesian modelling of GWAS data using linear mixed models</a>
      
      <a href="/post/2011-10-21/" class="list-group-item">William Astle: Bayesian modelling of GWAS data using linear mixed models</a>
      
      <a href="/post/2011-10-14/" class="list-group-item">Debbie Dupuis: Modeling non-stationary extremes: The case of heat waves | Richard A. Davis: Estimating extremal dependence in time series via the extremogram</a>
      
      <a href="/post/2011-10-7/" class="list-group-item">Nikolai Kolev: Nonexchangeability and radial asymmetry identification via bivariate quantiles, with financial applications</a>
      
      <a href="/post/2011-09-30/" class="list-group-item">Ioana A. Cosma: Data sketching for cardinality and entropy estimation?</a>
      
      <a href="/post/2011-09-23/" class="list-group-item">Shaowei Lin: What is singular learning theory?</a>
      
      <a href="/post/2011-09-16/" class="list-group-item">Elif F. Acar: Inference and model selection for pair-copula constructions</a>
      
      <a href="/post/2011-09-09/" class="list-group-item">Ed Susko: Properties of Bayesian posteriors and bootstrap support in phylogenetic inference | Aur√©lie Labbe: An integrated hierarchical Bayesian model for multivariate eQTL genetic mapping</a>
      
      <a href="/post/2011-08-31/" class="list-group-item">Johanna Ziegel: Precision estimation for stereological volumes</a>
      
      <a href="/tags/2011/" class="list-group-item">2011</a>
      
    </div>
  </section>

  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">CATEGORY</div>
    </div>
    <div class="list-group">
      
      <a href="/categories/mcgill-statistics-seminar" class="list-group-item">mcgill-statistics-seminar</a>
      
      <a href="/categories/crm-ism-gerad-colloque-de-statistique" class="list-group-item">crm-ism-gerad-colloque-de-statistique</a>
      
    </div>
  </section>
  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">TAG</div>
    </div>
    <div class="list-group">
      
      <a href="/tags/2011" class="list-group-item">2011</a>
      
    </div>
  </section>
  

</aside>


  </div>
</div>

      </div>
    </main>

    <footer class="l-footer">
      <div class="container">
        <p><span class="h-logo">&copy; My New Hugo Site</span></p>
        <aside>
          <p>Powered by <a href="https://gohugo.io/">Hugo</a>.</p>
          <p><a href="https://github.com/dim0627/hugo_theme_beg">Beg</a> designed by <a href="http://yet.unresolved.xyz/">Daisuke Tsuji</a>.</p>
        </aside>
      </div>
    </footer>

    <script src="//code.jquery.com/jquery-3.1.1.min.js"></script>
    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/highlight.min.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>
  </body>
</html>

