<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
<meta name="pinterest" content="nopin">
<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
<meta name="generator" content="Hugo 0.139.4">

/categories/mcgill-statistics-seminar/index.xml

<link rel="canonical" href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar/">


    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet">
    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/solarized_dark.min.css">
    <title>McGill Statistics Seminar - McGill Statistics Seminars</title>
    
    <link href="https://mcgillstat.github.io/css/styles.css" rel="stylesheet">
    

  </head>

  <body>
    
    
    

    <header class="l-header">
      <nav class="navbar navbar-default">
        <div class="container">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="https://mcgillstat.github.io/">McGill Statistics Seminars</a>
          </div>

          
          <div id="navbar" class="collapse navbar-collapse">
            
            <ul class="nav navbar-nav navbar-right">
              
              
              <li><a href="/">Current Seminar Series</a></li>
              
              
              
              <li><a href="/post/">Past Seminar Series</a></li>
              
              
            </ul>
            
          </div>
          

        </div>
      </nav>
    </header>

    <main>
      <div class="container">
        
<div class="row">
  <div class="col-md-8">

    
    <header class="page-header">
      <h1>McGill Statistics Seminar</h1>
    </header>
    

    <ul class="p-articles">
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-10-25T00:00:00JST">Oct 25, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019fall/2019-10-25/">Learning Connectivity Networks from High-Dimensional Point Processes</a></h2>
    <h3 class="post-meta">Ali Shojaie · Oct 25, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-10-25">Date: 2019-10-25</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burn-1205">Location: BURN 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>High-dimensional point processes have become ubiquitous in many scientific fields. For instance, neuroscientists use calcium florescent imaging to monitor the firing of thousands of neurons in live animals. In this talk, I will discuss new methodological, computational and theoretical developments for learning neuronal connectivity networks from high-dimensional point processes. Time permitting, I will also discuss a new approach for handling non-stationarity in high-dimensional time series.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019fall/2019-10-25/" title="Learning Connectivity Networks from High-Dimensional Point Processes">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-10-18T00:00:00JST">Oct 18, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019fall/2019-10-18/">Univariate and multivariate extremes of extendible random vectors</a></h2>
    <h3 class="post-meta">Klaus Herrmann · Oct 18, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-10-18">Date: 2019-10-18</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burn-1205">Location: BURN 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>In its most common form extreme value theory is concerned with the limiting distribution of location-scale transformed block-maxima $M_n = \max(X_1,\dots,X_n)$ of a sequence of identically distributed random variables $(X_i)$, $i\geq 1$.
In case the members of the sequence $(X_i)$ are independent, the weak
limiting behaviour of $M_n$ is adequately described by the classical Fisher-Tippett-Gnedenko theorem.
In this presentation we are interested in the case of dependent random variables $(X_i)$ while retaining a common marginal distribution function $F$ for all
$X_i$, $i\in\mathbb{N}$.
Complementary to the well established extreme value theory in a time series setting we consider a framework in which the dependence between (extreme) events does not decay over time.
This approach is facilitated by highlighting the connection between block-maxima and copula diagonals in an asymptotic context.
The main goal of this presentation is to discuss a generalization of the Fisher&ndash;Tippett&ndash;Gnedenko theorem in this setting, leading to limiting distributions that are not in the class of generalized extreme value distributions.
This result is exemplified for popular dependence structures related to extreme value, Archimedean and Archimax copulas.
Focusing on the class of hierarchical Archimedean copulas the results can further be extended to the multivariate setting.
Finally, we illustrate the resulting limit laws and discuss their properties.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019fall/2019-10-18/" title="Univariate and multivariate extremes of extendible random vectors">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-10-11T00:00:00JST">Oct 11, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019fall/2019-10-11/">Repulsiveness for integration (not my social program)</a></h2>
    <h3 class="post-meta">Jean-Francois Coeurjolly  · Oct 11, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-10-11">Date: 2019-10-11</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burn-1205">Location: BURN 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>Integral estimation in any dimension is an extensive topic, largely treated in the literature, with a broad range of applications. Monte-Carlo type methods arise naturally when one looks forward to quantifying/controlling the error. Many methods have already been developped: MCMC, Poisson disk sampling, QMC (and randomized versions), Bayesian quadrature, etc. In this talk, I&rsquo;ll consider a different approach which consists in defining the quadrature nodes as the realization of a spatial point process. In particular I&rsquo;ll show that a very specific class of determinantal point processes, a class of repulsive point patterns, has excellent properties and is able to estimate efficiently integrals for non-differentiable functions with an explicit and faster rate of convergence than current methods.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019fall/2019-10-11/" title="Repulsiveness for integration (not my social program)">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-09-27T00:00:00JST">Sep 27, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019fall/2019-09-27/">Regression Models for Spatial Images</a></h2>
    <h3 class="post-meta">Murray Clayton · Sep 27, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-09-27">Date: 2019-09-27</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-mcintyre-medical-building-room-521">Location: McIntyre Medical Building, Room 521</h4>
<h2 id="abstract">Abstract:</h2>
<p>This work is motivated by a problem in describing forest nitrogen
cycling, and a consequent goal of constructing regression models for
spatial images. Specifically, I present a functional concurrent linear
model (FLCM) with varying coefficients for two-dimensional spatial
images. To address overparameterization issues, the parameter surfaces
in this model are transformed into the wavelet domain and then sparse
representations are found using two different methods: LASSO and
Bayesian variable selection. I will briefly discuss extensions to
address missing data problems for colocated spatial images and the
modeling of tree species in landscape ecology. In addition I will
discuss the use of the sextant in marine navigation.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019fall/2019-09-27/" title="Regression Models for Spatial Images">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-09-20T00:00:00JST">Sep 20, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019fall/2019-09-20/">Deep Representation Learning using Discrete Domain Symmetries</a></h2>
    <h3 class="post-meta">Siamak Ravanbakhsh · Sep 20, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-09-20">Date: 2019-09-20</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burn-1205">Location: BURN 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>Symmetry has played a significant role in modern physics, in part by constraining the physical laws. I will discuss how it could play a fundamental role in AI by constraining the deep model design. In particular, I focus on discrete domain symmetries and through examples show how we can use this inductive bias as a principled means for constraining a feedforward layer and significantly improving its sample efficiency.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019fall/2019-09-20/" title="Deep Representation Learning using Discrete Domain Symmetries">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-09-13T00:00:00JST">Sep 13, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019fall/2019-09-13/">Integrative computational approach in genomics and healthcare</a></h2>
    <h3 class="post-meta">Yue Li · Sep 13, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-09-13">Date: 2019-09-13</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burn-1205">Location: BURN 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>In the current era of multi-omics and digital healthcare, we are facing unprecedented amount of data with tremendous opportunities to link molecular phenotypes with complex diseases. However, the lack of integrative statistical method hinders system-level interrogation of relevant disease-related pathways and the genetic implication in various healthcare outcome.</p>
<p>In this talk, I will present our current progress in mining genomics and healthcare data. In particular, I will cover two main topics: (1) a statistical approach to assess gene set enrichments using genetic and transcriptomic data; (2) multimodal latent topic model for mining electronic healthcare and whole genome sequencing data from small patient cohort.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019fall/2019-09-13/" title="Integrative computational approach in genomics and healthcare">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-09-06T00:00:00JST">Sep 6, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019fall/2019-09-06/">MAPLE; Semiparametric Estimation and Variable Selection for Length-biased Data with Heavy Censoring</a></h2>
    <h3 class="post-meta">Omidali Aghababaei Jazi · Sep 6, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-09-06">Date: 2019-09-06</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burn-1205">Location: BURN 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>In this talk, we discuss two problems of semiparametric estimation and variable selection for
length-biased data with heavy censoring.
The common feature of the proposed estimation procedures in the literature is that they only
put probability mass on failure times.
Under length-biased sampling, however, censoring is informative and failing to incorporate
censored observations into estimation can lead to
a substantial loss of efficiency. We propose two estimation procedures by computing the
likelihood contribution of both uncensored and censored observations.
For variable selection problem, we introduce a unified penalized estimating function and use an
optimization algorithm to solve it. We discuss
the asymptotic properties of the resulting penalized estimators. The work is motivated by the
International stroke Trial dataset collected in
Argentina in which the survival times of about 88% of the 545 cases are censored.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019fall/2019-09-06/" title="MAPLE; Semiparametric Estimation and Variable Selection for Length-biased Data with Heavy Censoring">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-04-26T00:00:00JST">Apr 26, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019winter/2019-04-26/">Graph Representation Learning and Applications</a></h2>
    <h3 class="post-meta">Jian Tang · Apr 26, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-04-26">Date: 2019-04-26</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burnside-1205">Location: BURNSIDE 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>Graphs, a general type of data structures for capturing interconnected objects, are ubiquitous in a variety of disciplines and domains ranging from computational social science, recommender systems, medicine, bioinformatics to chemistry. Representative examples of real-world graphs include social networks, user-item networks, protein-protein interaction networks, and molecular structures, which are represented as graphs. In this talk, I will introduce our work on learning effective representations of graphs such as learning low-dimensional node representations of large graphs (e.g., social networks, protein-protein interaction graphs, and knowledge graphs) and learning representations of entire graphs (e.g., molecule structures).</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019winter/2019-04-26/" title="Graph Representation Learning and Applications">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-04-12T00:00:00JST">Apr 12, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019winter/2019-04-12/">Estimating Time-Varying Causal Excursion Effect in Mobile Health with Binary Outcomes</a></h2>
    <h3 class="post-meta">Tianchen Qian · Apr 12, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-04-12">Date: 2019-04-12</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burnside-1205">Location: BURNSIDE 1205</h4>
<h2 id="abstract">Abstract:</h2>
<p>Advances in wearables and digital technology now make it possible to deliver behavioral, mobile health, interventions to individuals in their every-day life.  The micro-randomized trial (MRT) is increasingly used to provide data to inform the construction of these  interventions. This work is motivated by multiple MRTs that have been conducted or are currently in the field in which the primary outcome is a longitudinal binary outcome. The first, often called the primary, analysis in these trials is a marginal analysis that seeks to answer whether the data indicates that  a particular intervention component has an effect on the longitudinal binary outcome.  Under rather restrictive assumptions one can, based on existing literature, derive a semi-parametric, locally efficient estimator of the causal effect. In this talk, starting from this estimator, we develop multiple estimators that can be used as the basis of a primary analysis under more plausible assumptions. Simulation studies are conducted to compare the estimators.  We illustrate the developed methods using data from the MRT,  BariFit.  In BariFit, the goal is to support weight maintenance for individuals who received bariatric surgery.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019winter/2019-04-12/" title="Estimating Time-Varying Causal Excursion Effect in Mobile Health with Binary Outcomes">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2019-04-05T00:00:00JST">Apr 5, 2019</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2019winter/2019-04-05/">Bayesian Estimation of Individualized Treatment-Response Curves in Populations with Heterogeneous Treatment Effects</a></h2>
    <h3 class="post-meta">Yanxun Xu · Apr 5, 2019 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2019-04-05">Date: 2019-04-05</h4>
<h4 id="time-1530-1630">Time: 15:30-16:30</h4>
<h4 id="location-burnside-1104">Location: BURNSIDE 1104</h4>
<h2 id="abstract">Abstract:</h2>
<p>Estimating individual treatment effects is crucial for individualized or precision medicine. In reality, however, there is no way to obtain both the treated and untreated outcomes from the same person at the same time. An approximation can be obtained from randomized controlled trials (RCTs). Despite the limitations that randomizations are usually expensive, impractical or unethical, pre-specified variables may still not fully incorporate all the relevant characteristics capturing individual heterogeneity in treatment response. In this work, we use non-experimental data; we model heterogenous treatment effects in the studied population and provide a Bayesian estimator of the individual treatment response. More specifically, we develop a novel Bayesian nonparametric (BNP) method that leverages the G-computation formula to adjust for time-varying confounding in observational data, and it flexibly models sequential data to provide posterior inference over the treatment response at both group level and individual level. On a challenging dataset containing time series from patients admitted to intensive care unit (ICU), our approach reveals that these patients have heterogenous responses to the treatments used in managing kidney function. We also show that on held out data the resulting predicted outcome in response to treatment (or no treatment) is more accurate than alternative approaches.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2019winter/2019-04-05/" title="Bayesian Estimation of Individualized Treatment-Response Curves in Populations with Heterogeneous Treatment Effects">Read More…</a>
  </footer>
  
</article>
</li>
      
    </ul>

    
<nav>
  <ul class="pager">

    
    <li><a href="/categories/mcgill-statistics-seminar/page/10/">Previous</a></li>
    

    
    <li><a href="/categories/mcgill-statistics-seminar/page/12/">Next</a></li>
    

  </ul>
</nav>



  </div>
  <div class="col-md-4">
    
<aside class="l-sidebar">

  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">Recent Talks</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/tags/2025-fall/" class="list-group-item"> · Oct 24, 2025</a>
      
      <a href="https://mcgillstat.github.io/categories/" class="list-group-item"> · Oct 24, 2025</a>
      
      <a href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar/" class="list-group-item"> · Oct 24, 2025</a>
      
      <a href="https://mcgillstat.github.io/post/2025fall/2025-10-24/" class="list-group-item">Yang Feng · Oct 24, 2025</a>
      
      <a href="https://mcgillstat.github.io/tags/" class="list-group-item"> · Oct 24, 2025</a>
      
    </div>
  </section>

  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">CATEGORY</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar" class="list-group-item">mcgill statistics seminar</a>
      
      <a href="https://mcgillstat.github.io/categories/crm-ssc-prize-address" class="list-group-item">crm-ssc prize address</a>
      
      <a href="https://mcgillstat.github.io/categories/crm-colloquium" class="list-group-item">crm-colloquium</a>
      
    </div>
  </section>
  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">TAG</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/tags/2025-winter" class="list-group-item">2025 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2025-fall" class="list-group-item">2025 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2024-winter" class="list-group-item">2024 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2024-fall" class="list-group-item">2024 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-winter" class="list-group-item">2023 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-summer" class="list-group-item">2023 summer</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-fall" class="list-group-item">2023 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2022-winter" class="list-group-item">2022 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2022-fall" class="list-group-item">2022 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2021-winter" class="list-group-item">2021 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2021-fall" class="list-group-item">2021 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2020-winter" class="list-group-item">2020 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2020-fall" class="list-group-item">2020 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2019-winter" class="list-group-item">2019 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2019-fall" class="list-group-item">2019 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2018-winter" class="list-group-item">2018 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2018-fall" class="list-group-item">2018 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2017-winter" class="list-group-item">2017 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2017-fall" class="list-group-item">2017 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2016-winter" class="list-group-item">2016 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2016-fall" class="list-group-item">2016 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2015-winter" class="list-group-item">2015 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2015-fall" class="list-group-item">2015 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2014-winter" class="list-group-item">2014 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2014-fall" class="list-group-item">2014 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2013-winter" class="list-group-item">2013 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2013-fall" class="list-group-item">2013 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2012-winter" class="list-group-item">2012 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2012-fall" class="list-group-item">2012 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2011-fall" class="list-group-item">2011 fall</a>
      
    </div>
  </section>
  

</aside>



    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML"></script>
    
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [ ['$','$'], ["\\(","\\)"] ],
            displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
            processEscapes: true,
            processEnvironments: true
        },
        // Center justify equations in code and markdown cells. Elsewhere
        // we use CSS to left justify single line equations in code cells.
        displayAlign: 'center',
        "HTML-CSS": {
            styles: {'.MathJax_Display': {"margin": 0}},
            linebreaks: { automatic: true }
        }
    });
    </script>
    
  </div>
</div>

      </div>
    </main>

    <footer class="l-footer">
      <div class="container">
        <p><span class="h-logo">&copy; McGill Statistics Seminars</span></p>
        <aside>
          <p><a href="http://www.mcgill.ca/mathstat/">Department of Mathematics and Statistics</a>.</p>
          <p><a href="https://www.mcgill.ca/">McGill University</a></p>
        </aside>
      </div>
    </footer>

    <script src="//code.jquery.com/jquery-3.1.1.min.js"></script>
    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/highlight.min.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>
  </body>
</html>

