<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
<meta name="pinterest" content="nopin">
<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
<meta name="generator" content="Hugo 0.136.5">

/categories/mcgill-statistics-seminar/index.xml

<link rel="canonical" href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar/">


    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet">
    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/solarized_dark.min.css">
    <title>McGill Statistics Seminar - McGill Statistics Seminars</title>
    
    <link href="https://mcgillstat.github.io/css/styles.css" rel="stylesheet">
    

  </head>

  <body>
    
    
    

    <header class="l-header">
      <nav class="navbar navbar-default">
        <div class="container">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="https://mcgillstat.github.io/">McGill Statistics Seminars</a>
          </div>

          
          <div id="navbar" class="collapse navbar-collapse">
            
            <ul class="nav navbar-nav navbar-right">
              
              
              <li><a href="/">Current Seminar Series</a></li>
              
              
              
              <li><a href="/post/">Past Seminar Series</a></li>
              
              
            </ul>
            
          </div>
          

        </div>
      </nav>
    </header>

    <main>
      <div class="container">
        
<div class="row">
  <div class="col-md-8">

    
    <header class="page-header">
      <h1>McGill Statistics Seminar</h1>
    </header>
    

    <ul class="p-articles">
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-10-25T00:00:00JST">Oct 25, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024fall/2024-10-25/">New Statistical Methods and Quality Control for Industry 4.0</a></h2>
    <h3 class="post-meta">Antonio Lepore · Oct 25, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-10-25">Date: 2024-10-25</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-online-retransmitted-in-burnside-1104">Location: Online, retransmitted in Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj81908885431httpsmcgillzoomusj81908885431"><a href="https://mcgill.zoom.us/j/81908885431">https://mcgill.zoom.us/j/81908885431</a></h4>
<h4 id="meeting-id-819-0888-5431">Meeting ID: 819 0888 5431</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>Statistical and machine learning techniques have emerged as powerful tools in Industry 4.0 to benefit from the increasing availability of sensors and data and enable informed decision-making and process optimization. This seminar will provide an overview of several industrial applications in high-dimensional settings developed by the Statistics for Engineering Research (SFERe) group (<a href="https://www.sfere.unina.it">www.sfere.unina.it</a>), affiliated with the Department of Industrial Engineering at the University of Naples FEDERICO II. In these applications, the definition and use of novel statistical methods have been a competitive advantage for naval, automotive, and rail companies. The open-source R software packages implementing these methods will also be mentioned to highlight their accessibility and potential applicability in different industrial contexts.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024fall/2024-10-25/" title="New Statistical Methods and Quality Control for Industry 4.0">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-10-11T00:00:00JST">Oct 11, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024fall/2024-10-11/">A functional data approach for statistical shapes analysis</a></h2>
    <h3 class="post-meta">Issam Moindjié · Oct 11, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-10-11">Date: 2024-10-11</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-in-person-burnside-1104">Location: In person, Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj87824357176httpsmcgillzoomusj87824357176"><a href="https://mcgill.zoom.us/j/87824357176">https://mcgill.zoom.us/j/87824357176</a></h4>
<h4 id="meeting-id-878-2435-7176">Meeting ID: 878 2435 7176</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>The shape $\tilde{\mathbf{X}}$ of a random planar curve, $\mathbf{X}$, is what remains when the deformation variables (scaling, rotation, translation, and reparametrization) are removed. Previous studies in statistical shape analysis have focused on analyzing $\tilde{\bf X}$ through discrete observations of ${\bf X}$. While this approach has some computational advantages, it overlooks the continuous nature of variables: $\tilde{\bf X}$, ${\bf X}$, and it ignores the potential dependence of deformation variables on each other and $\tilde{ \bf X}$, which results in a loss of information in the data structure. I will introduce a new framework for studying $\bf X$ based on functional data analysis in this presentation. Basis expansion techniques are employed to find analytic solutions for deformation variables such as rotation and parametrization deformations. Then, the generative model of $\bf X$ is investigated using a joint-principal component analysis approach. Numerical experiments on synthetic and real datasets demonstrate how this new approach performs better at analyzing random planar curves than traditional functional data methods.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024fall/2024-10-11/" title="A functional data approach for statistical shapes analysis">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-09-27T00:00:00JST">Sep 27, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024fall/2024-09-27/">VCBART: Bayesian trees for varying coefficients</a></h2>
    <h3 class="post-meta">Ray Bai · Sep 27, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-09-27">Date: 2024-09-27</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-online-retransmitted-in-burnside-1104">Location: Online, retransmitted in Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj88350756970httpsmcgillzoomusj88350756970"><a href="https://mcgill.zoom.us/j/88350756970">https://mcgill.zoom.us/j/88350756970</a></h4>
<h4 id="meeting-id-883-5075-6970">Meeting ID: 883 5075 6970</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>The linear varying coefficient models posits a linear relationship between an outcome and covariates in which the covariate effects are modeled as functions of additional effect modifiers. Despite a long history of study and use in statistics and econometrics, state-of-the-art varying coefficient modeling methods cannot accommodate multivariate effect modifiers without imposing restrictive functional form assumptions or involving computationally intensive hyperparameter tuning. In response, we introduce VCBART which flexibly estimates the covariate effect in a varying coefficient model using Bayesian Additive Regression Trees. With simple default settings, VCBART outperforms existing varying coefficient methods in terms of covariate effect estimation, uncertainty quantification, and outcome prediction. Theoretically, we show that the VCBART posterior contracts at the near-minimax optimal rate. Finally, we illustrate the utility of VCBART through simulation studies and a real data application examining how the association between later-life cognition and measures of socioeconomic position vary with respect to age and sociodemographics.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024fall/2024-09-27/" title="VCBART: Bayesian trees for varying coefficients">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-09-20T00:00:00JST">Sep 20, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024fall/2024-09-20/">Variance reduction by occluding a Markov chain</a></h2>
    <h3 class="post-meta">Florian Maire · Sep 20, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-09-20">Date: 2024-09-20</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-in-person-burnside-1104">Location: In person, Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj88265323185httpsmcgillzoomusj88265323185"><a href="https://mcgill.zoom.us/j/88265323185">https://mcgill.zoom.us/j/88265323185</a></h4>
<h4 id="meeting-id-882-6532-3185">Meeting ID: 882 6532 3185</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>Stochastic algorithms which simulate random variables/processes on a computer to estimate intractable quantities are ubiquitous in Statistics and elsewhere. One such method is Markov chain Monte Carlo which, under mild conditions, offer asymptotical (in time) guarantees. In this talk, we define infinitely many stopping times at which an ergodic Markov chain is occluded by a (conditionally) independent process. The resulting process, called the occluded process, is not Markov, but provided that the stopping times/independent process are cleverly defined, we show that it is ergodic. One particularly powerful way to define the stopping times/independent process leverages the recent advances in ML regarding approximations of probability distributions (divergence minimization, normalizing flows, etc.). We discuss the variance reduction effect of the occluded process through some illustrations and (weak) theoretical results in some limiting regime.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024fall/2024-09-20/" title="Variance reduction by occluding a Markov chain">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-04-12T00:00:00JST">Apr 12, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024winter/2024-04-12/">Free energy fluctuations of spherical spin glasses near the critical temperature threshold</a></h2>
    <h3 class="post-meta">Elizabeth Collins-Woodfin · Apr 12, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-04-12">Date: 2024-04-12</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location--in-person-burnside-1104">Location:  In person, Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj86957985232httpsmcgillzoomusj86957985232"><a href="https://mcgill.zoom.us/j/86957985232">https://mcgill.zoom.us/j/86957985232</a></h4>
<h4 id="meeting-id-869-5798-5232">Meeting ID: 869 5798 5232</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>One of the fascinating phenomena of spin glasses is the dramatic change in behavior that occurs between the high and low temperature regimes.  In addition to its physical meaning, this phase transition corresponds to a detection threshold with respect to the signal-to-noise ratio in a spiked matrix model.  The free energy of the spherical Sherrington-Kirkpatrick (SSK) model has Gaussian fluctuations at high temperature, but Tracy-Widom fluctuations at low temperature.  A similar phenomenon holds for the bipartite SSK model, and we show that, when the temperature is within a small window around the critical temperature, the free energy fluctuations converge to an independent sum of Gaussian and Tracy-Widom random variables (joint work with Han Le).  Our work follows two recent papers that proved similar results for the SSK model (by Landon and by Johnstone, Klochkov, Onatski, Pavlyshyn).  From a statistical perspective, the free energy of SSK and bipartite SSK correspond to log-likelihood ratios for spiked Wigner and spiked Wishart matrices respectively.  Analyzing bipartite SSK at critical temperature requires a variety of tools including classical random matrix results, contour integral techniques, and a CLT for the log-characteristic polynomial of Wishart random matrices evaluated near the spectral edge.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024winter/2024-04-12/" title="Free energy fluctuations of spherical spin glasses near the critical temperature threshold">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-03-22T00:00:00JST">Mar 22, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024winter/2024-03-22/">Minimum Covariance Determinant: Spectral Embedding and Subset Size Determination</a></h2>
    <h3 class="post-meta">Qiang Heng · Mar 22, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-03-22">Date: 2024-03-22</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-online-retransmitted-in-burnside-1104">Location: Online, retransmitted in Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj81895414756httpsmcgillzoomusj81895414756"><a href="https://mcgill.zoom.us/j/81895414756">https://mcgill.zoom.us/j/81895414756</a></h4>
<h4 id="meeting-id-818-9541-4756">Meeting ID: 818 9541 4756</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>This paper introduces several enhancements to the minimum covariance determinant method of outlier detection and robust estimation of means and covariances. We leverage the principal component transform to achieve dimension reduction and ultimately better analyses. Our best subset selection algorithm strategically combines statistical depth and concentration steps. To ascertain the appropriate subset size and number of principal components, we introduce a bootstrap procedure that estimates the instability of the best subset algorithm. The parameter combination exhibiting minimal instability proves ideal for the purposes of outlier detection and robust estimation. Rigorous benchmarking against prominent MCD variants showcases our approach&rsquo;s superior statistical performance and computational speed in high dimensions. Application to a fruit spectra data set and a cancer genomics data set illustrates our claims.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024winter/2024-03-22/" title="Minimum Covariance Determinant: Spectral Embedding and Subset Size Determination">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-03-01T00:00:00JST">Mar 1, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024winter/2024-03-01/">Recent advances in causal inference under irregular and informative observation times for the outcome</a></h2>
    <h3 class="post-meta">Janie Coulombe · Mar 1, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-03-01">Date: 2024-03-01</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-in-person-burnside-1104">Location: In person, Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj89811237909httpsmcgillzoomusj89811237909"><a href="https://mcgill.zoom.us/j/89811237909">https://mcgill.zoom.us/j/89811237909</a></h4>
<h4 id="meeting-id-898-1123-7909">Meeting ID: 898 1123 7909</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>Electronic health records (EHR) data contain rich information about patients’ health condition, comorbidities, clinical outcomes, and drug prescriptions. They are often used to draw causal inferences and compare different treatments’ effectiveness. However, these data are not experimental. They present with special features that should be addressed or that may affect the inference. One of these features is the irregular observation of the longitudinal processes used in the inference. In longitudinal studies in which we seek the causal effect of a treatment on a repeated outcome, for instance, covariate-dependent observation of the outcome has been shown to bias standard causal estimators. In this presentation, I will review recent work and present some of the most interesting findings in this area of research. Themes will include identifiability, efficiency, and alternatives to weighting methods to address irregular observation times.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024winter/2024-03-01/" title="Recent advances in causal inference under irregular and informative observation times for the outcome">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-02-16T00:00:00JST">Feb 16, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024winter/2024-02-16/">Matrix completion in genetic methylation studies: LMCC, a Linear Model of Coregionalization with informative Covariates</a></h2>
    <h3 class="post-meta">Karim Oualkacha · Feb 16, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-02-16">Date: 2024-02-16</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-in-person-burnside-1104">Location: In person, Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj82678428848httpsmcgillzoomusj82678428848"><a href="https://mcgill.zoom.us/j/82678428848">https://mcgill.zoom.us/j/82678428848</a></h4>
<h4 id="meeting-id-826-7842-8848">Meeting ID: 826 7842 8848</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>DNA methylation is an important epigenetic mark that modulates gene expression through the inhibition of transcriptional proteins binding to DNA. As in many other omics experiments, missing values is an issue and appropriate imputation techniques are important to avoid an unnecessary sample size reduction as well as to optimally leverage the information collected. We consider the case where a relatively small number of samples are processed via an expensive high-density Whole Genome Bisulfite Sequencing (WGBS) strategy and a larger number of samples are processed using more affordable low-density array-based technologies. In such cases, one can impute/complete the data matrix of the low coverage (array-based) methylation data using the high-density information provided by the WGBS samples. In this work, we propose an efficient Linear Model of Coregionalization with informative Covariates (LMCC) to predict missing values based on observed values and informative covariates. Our model assumes that at each genomics position, the methylation vector of all samples is linked to the set of fixed factors (covariates) and a set of latent factors. Furthermore, we exploit the functional nature of the data and the spatial correlation across positions/sites by assuming Gaussian processes on the fixed and latent coefficient vectors, respectively. Our simulations show that the use of covariates can significantly improve the accuracy of imputed values, especially in cases where missing data contain some relevant information about the explanatory variable. We also show that the proposed model is efficient when the number of columns is much greater than the number of rows in the data matrix-which is usually the case in methylation data analysis. Finally, we apply and compare the proposed method with alternative approaches to complete a matrix of DNA methylation containing 15 rows (methylation samples) and 1 million columns (sites).
Joint work with Melina Ribaud and Aurelie Labbe (HEC, Montreal).</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024winter/2024-02-16/" title="Matrix completion in genetic methylation studies: LMCC, a Linear Model of Coregionalization with informative Covariates">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-02-09T00:00:00JST">Feb 9, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024winter/2024-02-09/">Mesoscale two-sample testing for networks</a></h2>
    <h3 class="post-meta">Peter William MacDonald · Feb 9, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-02-09">Date: 2024-02-09</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-in-person-burnside-1104">Location: In person, Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj87465663442httpsmcgillzoomusj87465663442"><a href="https://mcgill.zoom.us/j/87465663442">https://mcgill.zoom.us/j/87465663442</a></h4>
<h4 id="meeting-id-874-6566-3442">Meeting ID: 874 6566 3442</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>Networks arise naturally in many scientific fields as a representation of pairwise connections. Statistical network analysis has most often considered a single large network, but it is common in a number of applications, for example, neuroimaging, to observe multiple networks on a shared node set. When these networks are grouped by case-control status or another categorical covariate, the classical statistical question of two-sample comparison arises. In this work, we address the problem of testing for statistically significant differences in a prespecified subset of the connections. This general framework allows an analyst to focus on a single node, a specific region of interest, or compare whole networks. In this &ldquo;mesoscale&rdquo; setting, we develop statistically sound projection-based tests for two-sample comparison in both weighted and binary edge networks. Our approach can leverage all available network information, and learn informative projections which improve testing power when low-dimensional network structure is present.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024winter/2024-02-09/" title="Mesoscale two-sample testing for networks">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2024-02-02T00:00:00JST">Feb 2, 2024</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2024winter/2024-02-02/">Fast calibration of FARIMA models with dependent errors</a></h2>
    <h3 class="post-meta">Youssef Esstafa · Feb 2, 2024 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2024-02-02">Date: 2024-02-02</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="location-online-retransmitted-in-burnside-1104">Location: Online, retransmitted in Burnside 1104</h4>
<h4 id="httpsmcgillzoomusj89669635642httpsmcgillzoomusj89669635642"><a href="https://mcgill.zoom.us/j/89669635642">https://mcgill.zoom.us/j/89669635642</a></h4>
<h4 id="meeting-id-896-6963-5642">Meeting ID: 896 6963 5642</h4>
<h4 id="passcode-none">Passcode: None</h4>
<h2 id="abstract">Abstract:</h2>
<p>In this work, we investigate the asymptotic properties of Le Cam’s one-step estimator for weak Fractionally AutoRegressive Integrated Moving-Average (FARIMA) models. For these models, noises are uncorrelated but neither necessarily independent nor martingale differences errors. We show under some regularity assumptions that the one-step estimator is strongly consistent and asymptotically normal with the same asymptotic variance as the least squares estimator. We show through simulations that the proposed estimator reduces computational time compared with the least squares estimator.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2024winter/2024-02-02/" title="Fast calibration of FARIMA models with dependent errors">Read More…</a>
  </footer>
  
</article>
</li>
      
    </ul>

    
<nav>
  <ul class="pager">

    
    <li><a href="/categories/mcgill-statistics-seminar/">Previous</a></li>
    

    
    <li><a href="/categories/mcgill-statistics-seminar/page/3/">Next</a></li>
    

  </ul>
</nav>



  </div>
  <div class="col-md-4">
    
<aside class="l-sidebar">

  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">Recent Talks</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/tags/2025-winter/" class="list-group-item"> · Feb 28, 2025</a>
      
      <a href="https://mcgillstat.github.io/categories/" class="list-group-item"> · Feb 28, 2025</a>
      
      <a href="https://mcgillstat.github.io/post/2025winter/2025-02-28/" class="list-group-item">Johannes Lederer · Feb 28, 2025</a>
      
      <a href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar/" class="list-group-item"> · Feb 28, 2025</a>
      
      <a href="https://mcgillstat.github.io/tags/" class="list-group-item"> · Feb 28, 2025</a>
      
    </div>
  </section>

  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">CATEGORY</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar" class="list-group-item">mcgill statistics seminar</a>
      
      <a href="https://mcgillstat.github.io/categories/crm-ssc-prize-address" class="list-group-item">crm-ssc prize address</a>
      
      <a href="https://mcgillstat.github.io/categories/crm-colloquium" class="list-group-item">crm-colloquium</a>
      
    </div>
  </section>
  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">TAG</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/tags/2025-winter" class="list-group-item">2025 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2024-winter" class="list-group-item">2024 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2024-fall" class="list-group-item">2024 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-winter" class="list-group-item">2023 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-summer" class="list-group-item">2023 summer</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-fall" class="list-group-item">2023 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2022-winter" class="list-group-item">2022 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2022-fall" class="list-group-item">2022 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2021-winter" class="list-group-item">2021 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2021-fall" class="list-group-item">2021 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2020-winter" class="list-group-item">2020 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2020-fall" class="list-group-item">2020 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2019-winter" class="list-group-item">2019 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2019-fall" class="list-group-item">2019 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2018-winter" class="list-group-item">2018 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2018-fall" class="list-group-item">2018 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2017-winter" class="list-group-item">2017 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2017-fall" class="list-group-item">2017 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2016-winter" class="list-group-item">2016 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2016-fall" class="list-group-item">2016 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2015-winter" class="list-group-item">2015 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2015-fall" class="list-group-item">2015 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2014-winter" class="list-group-item">2014 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2014-fall" class="list-group-item">2014 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2013-winter" class="list-group-item">2013 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2013-fall" class="list-group-item">2013 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2012-winter" class="list-group-item">2012 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2012-fall" class="list-group-item">2012 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2011-fall" class="list-group-item">2011 fall</a>
      
    </div>
  </section>
  

</aside>



    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML"></script>
    
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [ ['$','$'], ["\\(","\\)"] ],
            displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
            processEscapes: true,
            processEnvironments: true
        },
        // Center justify equations in code and markdown cells. Elsewhere
        // we use CSS to left justify single line equations in code cells.
        displayAlign: 'center',
        "HTML-CSS": {
            styles: {'.MathJax_Display': {"margin": 0}},
            linebreaks: { automatic: true }
        }
    });
    </script>
    
  </div>
</div>

      </div>
    </main>

    <footer class="l-footer">
      <div class="container">
        <p><span class="h-logo">&copy; McGill Statistics Seminars</span></p>
        <aside>
          <p><a href="http://www.mcgill.ca/mathstat/">Department of Mathematics and Statistics</a>.</p>
          <p><a href="https://www.mcgill.ca/">McGill University</a></p>
        </aside>
      </div>
    </footer>

    <script src="//code.jquery.com/jquery-3.1.1.min.js"></script>
    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/highlight.min.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>
  </body>
</html>

