<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
<meta name="pinterest" content="nopin">
<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
<meta name="generator" content="Hugo 0.139.4">

/categories/mcgill-statistics-seminar/index.xml

<link rel="canonical" href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar/">


    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet">
    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/solarized_dark.min.css">
    <title>McGill Statistics Seminar - McGill Statistics Seminars</title>
    
    <link href="https://mcgillstat.github.io/css/styles.css" rel="stylesheet">
    

  </head>

  <body>
    
    
    

    <header class="l-header">
      <nav class="navbar navbar-default">
        <div class="container">
          <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
              <span class="sr-only">Toggle navigation</span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
              <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="https://mcgillstat.github.io/">McGill Statistics Seminars</a>
          </div>

          
          <div id="navbar" class="collapse navbar-collapse">
            
            <ul class="nav navbar-nav navbar-right">
              
              
              <li><a href="/">Current Seminar Series</a></li>
              
              
              
              <li><a href="/post/">Past Seminar Series</a></li>
              
              
            </ul>
            
          </div>
          

        </div>
      </nav>
    </header>

    <main>
      <div class="container">
        
<div class="row">
  <div class="col-md-8">

    
    <header class="page-header">
      <h1>McGill Statistics Seminar</h1>
    </header>
    

    <ul class="p-articles">
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-08-17T00:00:00JST">Aug 17, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-08-17/">Empirical Bayes Control of the False Discovery Exceedance</a></h2>
    <h3 class="post-meta">Pallavi Basu · Aug 17, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-08-17">Date: 2023-08-17</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="hybrid-in-person--zoom">Hybrid: In person / Zoom</h4>
<h4 id="location-burnside-hall-1104">Location: Burnside Hall 1104</h4>
<h4 id="httpsmcgillzoomusj89623344755pwds1e0qwvjsm8wrhdiyu5izzllsxnjut09httpsmcgillzoomusj89623344755pwds1e0qwvjsm8wrhdiyu5izzllsxnjut09"><a href="https://mcgill.zoom.us/j/89623344755?pwd=S1E0QWVjSm8wRHdIYU5IZzllSXNjUT09">https://mcgill.zoom.us/j/89623344755?pwd=S1E0QWVjSm8wRHdIYU5IZzllSXNjUT09</a></h4>
<h4 id="meeting-id-896-2334-4755">Meeting ID: 896 2334 4755</h4>
<h4 id="passcode-287381">Passcode: 287381</h4>
<h2 id="abstract">Abstract:</h2>
<p>In sparse large-scale testing problems where the false discovery proportion (FDP) is highly
variable, the false discovery exceedance (FDX) provides a valuable alternative to the
widely used false discovery rate (FDR). We develop an empirical Bayes approach to
controlling the FDX. We show that for independent hypotheses from a two-group model
and dependent hypotheses from a Gaussian model fulfilling the exchangeability condition,
an oracle decision rule based on ranking and thresholding the local false discovery rate
(lfdr) is optimal in the sense that the power is maximized subject to FDX constraint. We
propose a data-driven FDX procedure that emulates the oracle via carefully designed
computational shortcuts. We investigate the empirical performance of the proposed
method using simulations and illustrate the merits of FDX control through an application
for identifying abnormal stock trading strategies.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-08-17/" title="Empirical Bayes Control of the False Discovery Exceedance">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-08-14T00:00:00JST">Aug 14, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-08-14/">Residual-based estimation of parametric copulas under regression</a></h2>
    <h3 class="post-meta">Yue Zhao · Aug 14, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-08-14">Date: 2023-08-14</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="hybrid-in-person--zoom">Hybrid: In person / Zoom</h4>
<h4 id="location-burnside-hall-1104">Location: Burnside Hall 1104</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>We study a multivariate response regression model where each coordinate is described by a location-scale regression, and where the dependence structure of the &ldquo;noise&rdquo; terms in the regression is described by a parametric copula.  Our goal is to estimate the associated Euclidean copula parameter given a sample of the response and the covariate.  In the absence of the copula sample, the oracle ranks in the usual pseudo-likelihood estimation procedure are no longer computable.  Instead, we base our estimation on the residual ranks calculated from some preliminary estimators of the regression functions.  We show that the residual-based estimators are asymptotically equivalent to their oracle counterparts, even when the dimension of the covariate in the regression is moderately diverging.  Partially to serve this objective, we also study the weighted convergence of the residual empirical processes.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-08-14/" title="Residual-based estimation of parametric copulas under regression">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-03-24T00:00:00JST">Mar 24, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-03-24/">Confidence sets for Causal Discovery</a></h2>
    <h3 class="post-meta">Mladen Kolar · Mar 24, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-03-24">Date: 2023-03-24</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="on-zoom-only">On Zoom only</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>Causal discovery procedures are popular methods for discovering causal
structure across the physical, biological, and social sciences.
However, most procedures for causal discovery only output a single
estimated causal model or single equivalence class of models. We
propose a procedure for quantifying uncertainty in causal discovery.
Specifically, we consider linear structural equation models with
non-Gaussian errors and propose a procedure which returns a confidence
sets of causal orderings which are not ruled out by the data. We show
that asymptotically, the true causal ordering will be contained in the
returned set with some user specified probability.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-03-24/" title="Confidence sets for Causal Discovery">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-03-17T00:00:00JST">Mar 17, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-03-17/">Excursions in Statistical History: Highlights</a></h2>
    <h3 class="post-meta">James Hanley · Mar 17, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-03-17">Date: 2023-03-17</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="hybrid-in-person--zoom">Hybrid: In person / Zoom</h4>
<h4 id="location-burnside-hall-1104">Location: Burnside Hall 1104</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>Over the last 20 years, the speaker has delved into  the origins of &lsquo;regression&rsquo;;  the development of the &rsquo;t&rsquo; and &lsquo;Poisson&rsquo; distributions; forerunners of the &lsquo;hazard&rsquo; function; and the statistical design and conduct of US Selective Service lotteries from 1917 onwards.  This talk will recount the stories, data and simulations behind some of these, and provide some modern-day re-enactments.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-03-17/" title="Excursions in Statistical History: Highlights">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-03-10T00:00:00JST">Mar 10, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-03-10-2/">Heteroskedastic Sparse PCA in High Dimensions</a></h2>
    <h3 class="post-meta">Zhao Ren · Mar 10, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-03-10">Date: 2023-03-10</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="hybrid-in-person--zoom">Hybrid: In person / Zoom</h4>
<h4 id="location-burnside-hall-1104">Location: Burnside Hall 1104</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>Principal component analysis (PCA) is one of the most commonly used techniques for dimension reduction and feature extraction. Though it has been well-studied for high-dimensional sparse PCA, little is known when the noise is heteroskedastic, which turns out to be ubiquitous in many scenarios, like biological sequencing data and information network data. We propose an iterative algorithm for sparse PCA in the presence of heteroskedastic noise, which alternatively updates the estimates of the sparse eigenvectors using the power method with adaptive thresholding in one step, and imputes the diagonal values of the sample covariance matrix to reduce the estimation bias due to heteroskedasticity in the other step. Our procedure is computationally fast and provably optimal under the generalized spiked covariance model, assuming the leading eigenvectors are sparse. A comprehensive simulation study demonstrates its robustness and effectiveness in various settings.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-03-10-2/" title="Heteroskedastic Sparse PCA in High Dimensions">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-03-10T00:00:00JST">Mar 10, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-03-10-1/">High Dimensional Logistic Regression Under Network Dependence</a></h2>
    <h3 class="post-meta">Somabha Mukherjee · Mar 10, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-03-10">Date: 2023-03-10</h4>
<h4 id="time-1415-1515-montreal-time">Time: 14:15-15:15 (Montreal time)</h4>
<h4 id="hybrid-in-person--zoom">Hybrid: In person / Zoom</h4>
<h4 id="location-burnside-hall-1104">Location: Burnside Hall 1104</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>The classical formulation of logistic regression relies on the independent sampling assumption, which is often violated when the outcomes interact through an underlying network structure, such as over a temporal/spatial domain or on a social network. This necessitates the development of models that can simultaneously handle both the network peer-effect (arising from neighborhood interactions) and the effect of (possibly) high-dimensional covariates. In this talk, I will describe a framework for incorporating such dependencies in a high-dimensional logistic regression model by introducing a quadratic interaction term, as in the Ising model, designed to capture the pairwise interactions from the underlying network. The resulting model can also be viewed as an Ising model, where the node-dependent external fields linearly encode the high-dimensional covariates. We use a penalized maximum pseudo-likelihood method for estimating the network peer-effect and the effect of the covariates (the regression coefficients), which, in addition to handling the high-dimensionality of the parameters, conveniently avoids the computational intractability of the maximum likelihood approach. Our results imply that even under network dependence it is possible to consistently estimate the model parameters at the same rate as in classical (independent) logistic regression, when the true parameter is sparse and the underlying network is not too dense. Towards the end, I will talk about the rates of consistency of our proposed estimator for various natural graph ensembles, such as bounded degree graphs, sparse Erdos-Renyi random graphs, and stochastic block models, which follow as a consequence of our general results. This is a joint work with Ziang Niu, Sagnik Halder, Bhaswar Bhattacharya and George
Michailidis.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-03-10-1/" title="High Dimensional Logistic Regression Under Network Dependence">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-02-17T00:00:00JST">Feb 17, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-02-17/">Epidemic Forecasting using Delayed Time Embedding</a></h2>
    <h3 class="post-meta">Lam Ho · Feb 17, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-02-17">Date: 2023-02-17</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>Forecasting the future trajectory of an outbreak plays a crucial role in the mission of managing emerging infectious disease epidemics. Compartmental models, such as the Susceptible-Exposed-Infectious-Recovered (SEIR), are the most popular tools for this task. They have been used extensively to combat many infectious disease outbreaks including the current COVID-19 pandemic. One downside of these models is that they assume that the dynamics of an epidemic follow a pre-defined dynamical system which may not capture the true trajectories of an outbreak. Consequently, the users need to make several modifications throughout an epidemic to ensure their models fit well with the data. However, there is no guarantee that these modifications can also help increase the precision of forecasting. In this talk, I will introduce a new method for predicting epidemics that does not make any assumption on the underlying dynamical system. Our method combines sparse random feature expansion and delay embedding to learn the trajectory of an epidemic.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-02-17/" title="Epidemic Forecasting using Delayed Time Embedding">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-02-10T00:00:00JST">Feb 10, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-02-10/">Efficient Label Shift Adaptation through the Lens of Semiparametric Models</a></h2>
    <h3 class="post-meta">Qinglong Tian · Feb 10, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-02-10">Date: 2023-02-10</h4>
<h4 id="time-1500-1600-montreal-time">Time: 15:00-16:00 (Montreal time)</h4>
<h4 id="hybrid-in-person--zoom">Hybrid: In person / Zoom</h4>
<h4 id="location-burnside-hall-1205">Location: Burnside Hall 1205</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>We study the domain adaptation problem with label shift in this work. Under the label shift context, the marginal distribution of the label varies across the training and testing datasets, while the conditional distribution of features given the label is the same. Traditional label shift adaptation methods either suffer from large estimation errors or require cumbersome post-prediction calibrations. To address these issues, we first propose a moment-matching framework for adapting the label shift based on the geometry of the influence function. Under such a framework, we propose a novel method named efficient label shift adaptation (ELSA), in which the adaptation weights can be estimated by solving linear systems. Theoretically, the ELSA estimator is root-n consistent (n is the sample size of the source data) and asymptotically normal. Empirically, we show that ELSA can achieve state-of-the-art estimation performances without post-prediction calibrations, thus, gaining computational efficiency.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-02-10/" title="Efficient Label Shift Adaptation through the Lens of Semiparametric Models">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-02-03T00:00:00JST">Feb 3, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-02-03/">Learning from a Biased Sample</a></h2>
    <h3 class="post-meta">Lihua Lei · Feb 3, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-02-03">Date: 2023-02-03</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>The empirical risk minimization approach to data-driven decision making assumes that we can learn a decision rule from training data drawn under the same conditions as the ones we want to deploy it under. However, in a number of settings, we may be concerned that our training sample is biased, and that some groups (characterized by either observable or unobservable attributes) may be under- or over-represented relative to the general population; and in this setting empirical risk minimization over the training set may fail to yield rules that perform well at deployment. Building on concepts from distributionally robust optimization and sensitivity analysis, we propose a method for learning a decision rule that minimizes the worst-case risk incurred under a family of test distributions whose conditional distributions of outcomes  given covariates  differ from the conditional training distribution by at most a constant factor, and whose covariate distributions are absolutely continuous with respect to the covariate distribution of the training data. We apply a result of Rockafellar and Uryasev to show that this problem is equivalent to an augmented convex risk minimization problem. We give statistical guarantees for learning a robust model using the method of sieves and propose a deep learning algorithm whose loss function captures our robustness target. We empirically validate our proposed method in simulations and a case study with the MIMIC-III dataset.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-02-03/" title="Learning from a Biased Sample">Read More…</a>
  </footer>
  
</article>
</li>
      
      <li><article class="li">
  <header>
    <ul class="p-facts">
      <li><i class="fa fa-calendar" aria-hidden="true"></i><time datetime="2023-01-20T00:00:00JST">Jan 20, 2023</time></li>
      <li><i class="fa fa-bookmark" aria-hidden="true"></i><a href="https://mcgillstat.github.io/post/">post</a></li>
      
    </ul>
    <h2 class="title"><a href="https://mcgillstat.github.io/post/2023winter/2023-01-20/">What is TWAS and how do we use it in integrating gene expression data</a></h2>
    <h3 class="post-meta">Quan Long · Jan 20, 2023 </h3>
  </header>

  
  <div class="summary"><h4 id="date-2023-01-20">Date: 2023-01-20</h4>
<h4 id="time-1530-1630-montreal-time">Time: 15:30-16:30 (Montreal time)</h4>
<h4 id="httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09httpsmcgillzoomusj83436686293pwdb0rmwmlxrxe3owr6nlnicwf5d0djqt09"><a href="https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09">https://mcgill.zoom.us/j/83436686293?pwd=b0RmWmlXRXE3OWR6NlNIcWF5d0dJQT09</a></h4>
<h4 id="meeting-id-834-3668-6293">Meeting ID: 834 3668 6293</h4>
<h4 id="passcode-12345">Passcode: 12345</h4>
<h2 id="abstract">Abstract:</h2>
<p>The transcriptome-wide association studies (TWAS) is a pioneering approach utilizing gene expression data to identify genetic basis of complex diseases. Its core component is called &ldquo;genetically regulated expression (GReX)&rdquo;. GReX links gene expression information with phenotype by serving as both the outcome of genotype-based expression models and the predictor for downstream association testing. Although it is popular and has been used in many high-profile projects, its mathematical nature and interpretation haven’t been rigorously verified. As such, we have first conducted power analysis using NCP-based closed forms (Cao et al, PLoS Genet 2021), based on which we realized that the common interpretation of TWAS that looks biologically sensible is actually mathematically questionable. Following this insight, by real data analysis and simulations, we demonstrated that current linear models of GReX inadvertently combine two separable steps of machine learning - feature selection and aggregation - which can be independently replaced to improve overall power (Cao et al, Genetics 2021). Based on this new interpretation, we have developed novel protocols disentangling feature selections and aggregations, leading to improved power and novel biological discoveries (Cao et al, BiB 2021; Genetics 2021). To promote this new understanding, we moved forward to develop two statistical tools utilizing gene expressions in identifying genetic basis of gene-gene interactions (Kossinna et al, in preparation) and low-effect genetic variants (Li et al, in review), respectively. Looking forward, our mathematical characterization of TWAS opens a door for a new way to integrate gene expressions in genetic studies towards the realization of precision medicine.</p></div>

  
  <footer>
    <a href="https://mcgillstat.github.io/post/2023winter/2023-01-20/" title="What is TWAS and how do we use it in integrating gene expression data">Read More…</a>
  </footer>
  
</article>
</li>
      
    </ul>

    
<nav>
  <ul class="pager">

    
    <li><a href="/categories/mcgill-statistics-seminar/page/3/">Previous</a></li>
    

    
    <li><a href="/categories/mcgill-statistics-seminar/page/5/">Next</a></li>
    

  </ul>
</nav>



  </div>
  <div class="col-md-4">
    
<aside class="l-sidebar">

  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">Recent Talks</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/tags/2025-winter/" class="list-group-item"> · Feb 7, 2025</a>
      
      <a href="https://mcgillstat.github.io/categories/" class="list-group-item"> · Feb 7, 2025</a>
      
      <a href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar/" class="list-group-item"> · Feb 7, 2025</a>
      
      <a href="https://mcgillstat.github.io/tags/" class="list-group-item"> · Feb 7, 2025</a>
      
      <a href="https://mcgillstat.github.io/post/2025winter/2025-02-07/" class="list-group-item">Michaël Lalancette · Feb 7, 2025</a>
      
    </div>
  </section>

  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">CATEGORY</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/categories/mcgill-statistics-seminar" class="list-group-item">mcgill statistics seminar</a>
      
      <a href="https://mcgillstat.github.io/categories/crm-ssc-prize-address" class="list-group-item">crm-ssc prize address</a>
      
      <a href="https://mcgillstat.github.io/categories/crm-colloquium" class="list-group-item">crm-colloquium</a>
      
    </div>
  </section>
  
  <section class="panel panel-default">
    <div class="panel-heading">
      <div class="panel-title">TAG</div>
    </div>
    <div class="list-group">
      
      <a href="https://mcgillstat.github.io/tags/2025-winter" class="list-group-item">2025 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2024-winter" class="list-group-item">2024 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2024-fall" class="list-group-item">2024 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-winter" class="list-group-item">2023 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-summer" class="list-group-item">2023 summer</a>
      
      <a href="https://mcgillstat.github.io/tags/2023-fall" class="list-group-item">2023 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2022-winter" class="list-group-item">2022 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2022-fall" class="list-group-item">2022 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2021-winter" class="list-group-item">2021 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2021-fall" class="list-group-item">2021 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2020-winter" class="list-group-item">2020 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2020-fall" class="list-group-item">2020 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2019-winter" class="list-group-item">2019 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2019-fall" class="list-group-item">2019 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2018-winter" class="list-group-item">2018 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2018-fall" class="list-group-item">2018 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2017-winter" class="list-group-item">2017 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2017-fall" class="list-group-item">2017 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2016-winter" class="list-group-item">2016 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2016-fall" class="list-group-item">2016 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2015-winter" class="list-group-item">2015 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2015-fall" class="list-group-item">2015 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2014-winter" class="list-group-item">2014 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2014-fall" class="list-group-item">2014 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2013-winter" class="list-group-item">2013 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2013-fall" class="list-group-item">2013 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2012-winter" class="list-group-item">2012 winter</a>
      
      <a href="https://mcgillstat.github.io/tags/2012-fall" class="list-group-item">2012 fall</a>
      
      <a href="https://mcgillstat.github.io/tags/2011-fall" class="list-group-item">2011 fall</a>
      
    </div>
  </section>
  

</aside>



    <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML"></script>
    
    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
        tex2jax: {
            inlineMath: [ ['$','$'], ["\\(","\\)"] ],
            displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
            processEscapes: true,
            processEnvironments: true
        },
        // Center justify equations in code and markdown cells. Elsewhere
        // we use CSS to left justify single line equations in code cells.
        displayAlign: 'center',
        "HTML-CSS": {
            styles: {'.MathJax_Display': {"margin": 0}},
            linebreaks: { automatic: true }
        }
    });
    </script>
    
  </div>
</div>

      </div>
    </main>

    <footer class="l-footer">
      <div class="container">
        <p><span class="h-logo">&copy; McGill Statistics Seminars</span></p>
        <aside>
          <p><a href="http://www.mcgill.ca/mathstat/">Department of Mathematics and Statistics</a>.</p>
          <p><a href="https://www.mcgill.ca/">McGill University</a></p>
        </aside>
      </div>
    </footer>

    <script src="//code.jquery.com/jquery-3.1.1.min.js"></script>
    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/highlight.min.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>
  </body>
</html>

